dataform/examples/dataform_udf_unit_test/README.md

```diff
@@ -0,0 +1,87 @@
+# Dataform UDF Unit Testing Example
+
+## How to run this example
+
+1. Clone this repo
+    1. **Automatic Clone**: Click below\
+       [![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fdanieldeleo%2Fbigquery-utils.git&cloudshell_workspace=dataform/examples/dataform_udf_unit_test&cloudshell_tutorial=tutorial.md&cloudshell_git_branch=dataform_examples&cloudshell_open_in_editor=definitions/test_cases.js) \
+       (clicking will automatically clone this repo in your Cloud Shell, open
+       the dataform udf testing example directory, and launch a tutorial)
+
+    1. **Manual Clone**: If you are not using the above link and instead
+       manually cloning, make sure to change into the correct directory as shown
+       below:
+       ```bash
+        git clone https://github.com/GoogleCloudPlatform/bigquery-utils.git
+        cd bigquery-utils/dataform/examples/dataform_udf_unit_test
+       ```
+1. Install Dataform CLI tool
+    ```bash
+    npm i -g @dataform/cli && dataform install
+    ```
+1. Generate the Dataform credentials file by running the following:
+   ```bash
+   dataform init-creds bigquery
+   ```
+1. Execute the unit tests by running the following:
+    ```bash
+    dataform test
+    ```
+
+## How to modify the example for your own UDFs
+
+1. Create environment variables for the path to your Dataform directory and your
```

Comment:
This line should be: 

### 1. While you’re still in the dataform_udf_unit_test directory, create environment variables for the name of your Dataform directory and your BigQuery project ID: